### PR TITLE
Supplant the "starter bug" term by "good first issue"

### PR DESCRIPTION
**Pull request by AnthonyLatsis**

### CODE REVIEW

**Review by shahmishal — APPROVED** [review]
These are great improvements!

**Comment by shahmishal** [comment]
@swift-ci test

File: contributing/_good_first_issues.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
-## Starter Bugs
+## Good first issues
```
Comment (alexandersandberg) [inline contributing/_good_first_issues.md:1]: I see that this section is currently not even displayed on the /contributing page. We should find a place for it.

File: contributing/_good_first_issues.md [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
+in some part of Swift, learn more about it, and make a real contribution.
 
-The current list of starter bugs is [maintained in the bug tracker][starter-bugs].
 
 {% comment %}
     // TODO: This is content I'd like to migrate into Jira behind a "starter" label of some sort. For now:
 
 * Swift Intermediate Language (SIL) round-tripping: make sure that the SIL parser can parse what the SIL printer prints. This is a great project for getting a feel for SIL and how it's used, and making it round-trippable has huge benefits for anyone working on the Swift compiler.
 * Warning control: [Clang](http://clang.llvm.org) has a great scheme for placing warnings into specific warning groups, allowing one to control (from the command line) which warnings are emitted by the compiler or are treated as errors. Swift needs that!
 {% endcomment %}
```
Comment (alexandersandberg) [inline contributing/_good_first_issues.md:18]: Maybe we can remove this comment as well.

File: contributing/_good_first_issues.md (hunk 1 above)
Comment (AnthonyLatsis) [reply contributing/_good_first_issues.md:1]: Yeah, going to bring this up at the next meeting of our workgroup.

File: contributing/_good_first_issues.md (hunk 2 above)
Comment (AnthonyLatsis) [reply contributing/_good_first_issues.md:18]: We should create issues for these first.

File: contributing/_good_first_issues.md (hunk 1 above)
Comment (kaishin) [reply contributing/_good_first_issues.md:1]:
```suggestion
## Good First Issues
```
To remain consistent with other `h2` headers.

**Comment by hborla** [comment]
@swift-ci please test

**Review by kaishin — APPROVED** [review]

**Pull request merged**